.github/workflows/ci.yml

```diff
@@ -12,7 +12,33 @@ env:
   RUST_FEATURES: "rodio-backend,lyric-finder,media-control,image,notify,clipboard"
 
 jobs:
-  rust-ci:
+   msrv:
+    if: github.event.pull_request.draft != true
+    strategy:
+      fail-fast: false
```

Comment:
why not `fail-fast` on this one?

Reply:
I think that being a `check` rather than `build` or `test`, the job should be fast enough to not matter for a few seconds, and adds the feedback on whether it broke for every platform or not - likely would be a platform dependant dependency. 
Though if keeping the MSRV for builds with all (compatible) features it would have to be bumped anyway.